Room/app/src/main/java/com/example/room/data/local/FriendRepositoryImpl.kt

```diff
@@ -0,0 +1,37 @@
+package com.example.room.data.local
+
+import com.example.room.data.local.datasource.LocalFriendDataSource
+import com.example.room.data.models.types.MBTI
+import com.example.room.data.models.types.MBTIFeatures
+import com.example.room.data.local.db.Friend
+import com.example.room.domain.repository.FriendRepository
+import kotlinx.coroutines.flow.Flow
+
+class FriendRepositoryImpl(
+    private val friendLocalFriendDataSource: LocalFriendDataSource
+) : FriendRepository {
+
+    override suspend fun insertFriend(friend: Friend): Long {
+        return friendLocalFriendDataSource.insertFriendToDB(friend)
+    }
+
+    override suspend fun updateFriend(friend: Friend): Int {
+        return friendLocalFriendDataSource.updateFriendToDB(friend)
+    }
+
+    override suspend fun deleteFriend(friend: Friend): Int {
+        return friendLocalFriendDataSource.deleteFriendFromDB(friend)
+    }
+
+    override suspend fun deleteAll(): Int {
```

Comment:
```suggestion
    override suspend fun deleteAllFriends(): Int {
```
통일성을 위해 함수명 뒤에 Friends를 붙여보는 건 어떨까요??ㅎㅎ

Reply:
너무 좋습니다!!